core/src/saros/communication/extensions/VersionExchangeExtension.java

```diff
@@ -1,54 +0,0 @@
-package saros.communication.extensions;
```

Comment:
yes this whole pr breaks compatibility, but @tobous, @m273d15 and i are fine with that

older clients will newer think about sending this packet because, as the `saros` feature is removed from service discovery, they will not see saros support.